packages/controller/src/main.ts

```diff
@@ -3032,18 +3032,43 @@ async function processMessage(msg: ioBroker.SendableMessage): Promise<null | voi
                         if (msg.message.link) {
                             if (!error && base64) {
                                 const buff = Buffer.from(base64, 'base64');
-                                states!.setBinaryState(`${hostObjectPrefix}.zip.${msg.message.link}`, buff, err => {
-                                    if (err) {
-                                        sendTo(msg.from, msg.command, { error: err }, msg.callback);
-                                    } else {
-                                        sendTo(
-                                            msg.from,
-                                            msg.command,
-                                            `${hostObjectPrefix}.zip.${msg.message.link}`,
-                                            msg.callback
-                                        );
-                                    }
-                                });
+                                if (msg.message.storeToFile) {
```

Comment:
I would love to porpose a rrename ofthis to eg "fileStorageNamespace" or such ... so that the property name clearly states that it is about a namespace ( I think we call it that way, or?). The name "storeToFile" more sounds like a boolean :-)

Reply:
Renamed